repo-pr-stats.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+from datetime import datetime
+import getopt
+import urllib
+import sys
+
+import requests
+
+
+def _parse_github_datetime(datetime_string):
+    try:
+        return datetime.strptime(datetime_string, '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
I'm impressed that this standard ISO format isn't part of the stdlib without manually defining it.

Probably not worth putting a lot more research into I suppose